sms_email_form/README.md

```diff
@@ -0,0 +1,14 @@
+# SMS + Email Form Template
+
+This HTML template will let you request email addresses and phone numbers from users and create new SMS and Email players out of them. You will need to add your Onesignal App ID into this template at the appropriate place.
+
+###### SMS Email form on Android
+![Screenshot](readme_assets/sms_email_form.png)
+
+## Safe Area (notch) Detection
+Certain mobile devices may have safe areas where operating system icons or a camera notch might exist. Our Onesignal SDK detects these areas and exposes them to the iam's HTML document via `safe-area-inset-*` CSS variables :point_down:
```

Comment:
do we have these variables exposed anywhere else in the documentation? i.e. in the custom html docs in our main docs?